corehq/apps/users/models.py

```diff
@@ -2342,6 +2342,11 @@ def get_device(self, device_id):
             if device.device_id == device_id:
                 return device
 
+    def get_devices_fcm_token(self):
+        if not self.devices:
+            return None
```

Comment:
I believe we should avoid mixing None with a list return. Better to return an empty list, so just having the list comprehensions seems okay here.

Reply:
Resolved in f81ef437a0c8d3971ffe3217c42ce98db6b35e93